src/otx/core/metrics/fmeasure.py

```diff
@@ -644,6 +644,7 @@ class FMeasure(Metric):
     def __init__(
         self,
         label_info: LabelInfo,
+        *,
```

Comment:
Why is this needed?

Reply:
I'd like to make sure that only `label_info` is an unique positional argument: https://realpython.com/python-asterisk-and-slash-special-parameters/#in-short-the-python-asterisk-and-slash-control-how-to-pass-values-to-functions

It does not affect to the runtime behavior.